README.md

```diff
@@ -181,3 +181,150 @@ toggles output to the terminal on or off.
 - `true` will show detailed information on failing assertions.
   This is the default, and best for using `true-cli`.
 - `false` to turn off all terminal output.
+
+## API
+
+### Tests
+
+Tests help define what we are testing for. They can be used individually or
+grouped as a test module.
+
+#### `@include test-module()`
+
+Defines a test module. Allows the grouping of several common purposed tests.
+
+- `@param` {`String`} `$description` [''] - Module description
+
+```scss
+  @include test-module('Description of the module being tested') {
+    // Your tests here
+  }
+```
+
+#### `@include test()`
+
+Defines a test. Allows the definition of a single test for your module.
+
+- `@param` {`String`} `$description` [''] - Test description
+
+```scss
+  @include test-module('My test module description') {
+    @include test('Your test description') {
+      // Your asserts go here.
+    }
+  }
+```
+
+### Asserts
+
+Asserts help define what we are testing for. A test must contain at least one
+assert but can have as many as necessary based on your testing needs.
+
+#### `@include assert()`
+
+Defines a single assert. Use `output()` and `expect()` mixins to define blocks
+for comparison.
+
+```scss
+  @include test('Your test description') {
+    @include assert('Your assert description') {
+      // We will test for an mixin input and output here
+    }
+  }
+```
+
+#### `@include output`
+
+Describe the test content to be evaluated
+
+```scss
+  @include test('Your test description') {
+    @include assert('Your assert description') {
+      @include output {
+        @include the-mixin-to-be-tested();
+      }
+    }
+  }
+```
+
+#### `@include expect`
+
+Describe the output content to be expected.
+
+```scss
+  @include test('Your test description') {
+    @include assert('Your assert description') {
+      @include output {
+        @include the-mixin-to-be-tested();
+      }
+
+      @include expect {
+        // The expected output of the-mixin-to-be-tested()
+      }
+    }
+  }
+```
+
+#### `@include assert-true()`
+
+Asserts that the output of a test is `true`.
+
+- `@param` {`*`} `$assert` - Assert to be tested
+- `@param` {`String`} `$description` [''] - Assert description
+
+```scss
+  @include test('Your test description') {
+    $test: sample-function(5);
+
+    @include assert-true($test, 'Your assert description');
+  }
+```
+
+#### `@include assert-false()`
+
+Asserts that the output of a test is `false`.
```

Comment:
`null` also returns falsey for this assertion. Empty strings and empty lists do not, but maybe they should? We could add an issue to fix that.